tests/GridFS/StreamWrapperFunctionalTest.php

```diff
@@ -173,25 +172,19 @@ public function testWritableStreamStatBeforeSaving()
 
     public function testWritableStreamStatAfterSaving()
     {
-        $id = new ObjectId;
-        $stream = $this->bucket->openUploadStream('filename', ['chunkSizeBytes' => 1024, '_id' => $id]);
-
-        $this->assertSame(6, fwrite($stream, 'foobar'));
-        $this->assertTrue(fclose($stream));
-        $currentTimestamp = time();
-        $stream = $this->bucket->openDownloadStream($id);
+        $stream = $this->bucket->openDownloadStream('length-10');
 
         $stat = fstat($stream);
         $this->assertSame(0100444, $stat[2]);
         $this->assertSame(0100444, $stat['mode']);
-        $this->assertSame(6, $stat[7]);
-        $this->assertSame(6, $stat['size']);
-        $this->assertGreaterThanOrEqual($currentTimestamp, $stat[9]);
-        $this->assertGreaterThanOrEqual($currentTimestamp, $stat['mtime']);
-        $this->assertGreaterThanOrEqual($currentTimestamp, $stat[10]);
-        $this->assertGreaterThanOrEqual($currentTimestamp, $stat['ctime']);
-        $this->assertSame(1024, $stat[11]);
-        $this->assertSame(1024, $stat['blksize']);
+        $this->assertSame(10, $stat[7]);
+        $this->assertSame(10, $stat['size']);
+        $this->assertSame(1484202200000/1000, $stat[9]);
```

Comment:
On the off chance this breaks 32-bit environments, let's just use `1484202200` here.